domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -3221,7 +3322,7 @@ public void selectSingle(AsyncResult<SQLConnection> conn, String sql, Tuple para
    * @param chunkSize cursor fetch size
    */
   public Future<Void> selectStream(String sql, Tuple params, int chunkSize, Handler<RowStream<Row>> rowStreamHandler) {
-    return withTrans(trans -> trans.selectStream(sql, params, chunkSize, rowStreamHandler));
+    return withReadTrans(trans -> trans.selectStream(sql, params, chunkSize, rowStreamHandler));
```

Comment:
The javadoc in Conn.java should be reviewed to show read methods where appropriate.

Reply:
> The javadoc in Conn.java should be reviewed to show read methods where appropriate.
I don't think I understand this comment. No methods in Conn.java was updated.

Reply:
The javadoc in Conn.java should mention read-only transactions where appropriate.
```
   * Sample usage:
   *
   * <pre>
   * postgresClient.withTrans(conn -> step1()
   *     .compose(x -> conn.selectStream("SELECT i FROM numbers WHERE i > $1", Tuple.tuple(5),
   *         rowStream -> rowStream.handler(row -> task.process(row))))
   *     .compose(x -> ...
   * </pre>
   *
   * <p>Use withReadTrans if all SQL queries are read-only (no nextval(), no UPDATE, ...):
   *
   * <pre>
   * postgresClient.withReadTrans(conn ->
   *     conn.selectStream("SELECT i FROM numbers WHERE i > $1", Tuple.tuple(5),
   *         rowStream -> rowStream.handler(row -> task.process(row))));
   * </pre>
```